lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -30,9 +30,46 @@ def _config_str_to_list(self, list_str):
 
         return user_list
 
+    def _parse_app_id(self, exec_task):
+        app_id = ''
+        if 'app_id' in exec_task['job_input']:
+            app_id = exec_task['job_input']['app_id']
+            app_id = app_id.replace('.', '/')
```

Comment:
just do `app_id = exec_task['job_input']['app_id'].replace('.', '/')`

Reply:
did so now.

Reply:
This is not updated.

Reply:
'app_id' may not always exist in exec_task['job_input'].

Reply:
@qzzhang look at above comments. It has nothing to do with 'app_id' existence.